guides/saas/getting-started.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Getting started with Coder's hosted beta
+description: Get started with a hosted Coder deployment.
+---
+
+To help you get started, we've put together this guide to help you understand
+how the Coder hosted beta works and how to access your deployment.
+
+## What is Coder hosted beta?
```

Comment:
@CoderErik created a lovely graphic demonstrating how this works. thoughts on adding a version of this in a later revision?

We also discussed adding a second graphic to demonstrate how the clients actually have a point-to-point connection with workspaces. Thoughts on that as well?

@jmilling33 could probably work his magic on it

[Shared Hosting.pdf](https://github.com/cdr/docs/files/7078351/Shared.Hosting.pdf)

Reply:
whoops here is the updated graphic

[Shared Hosting (1).pdf](https://github.com/cdr/docs/files/7078391/Shared.Hosting.1.pdf)

Reply:
Definitely think @CoderErik's graphic would fit well in here.

Reply:
Here's a link to png version
https://lucid.app/publicSegments/view/7e950228-6b5a-47eb-b9d4-ed5a82e5898d/image.png

Reply:
@jmilling33 from a design standpoint, should we defer to you on that? @CoderErik's graphic is accurate, but I just want to make sure we keeps things consistent.

Reply:
I am happy to restyle the graphic to make sure it's consistent with our branding. When do you need it by?

Reply:
@jmilling33 Just shared the original file with you. Check email for the link.

Reply:
@jmilling33 no hard deadline. It would be nice to update this doc in ~1 week or so via another PR.

For now, we can leave the illustration out